components/ethernet/Kconfig

```diff
@@ -10,7 +10,7 @@ config DMA_RX_BUF_NUM
         Larger number of buffers increases throughput.
         If enable flow ctrl, the num must be above 9 .
 
-config DMA_TX_BUF_NUM
+config ETH_DMA_TX_BUF_NUM
```

Comment:
While I agree that prefixing the configuration variables is a good idea, renaming existing ones will result in loss of configuration for project which had these vars set to non-default values.

Reply:
Ok, I'll revert this change.